packages/e2e/features/ui/components/authenticator/sign-in-force-new-password.feature

```diff
@@ -24,6 +24,12 @@ Feature: Sign In with Force New Password flow
     And I type my password
     And I click the "Sign in" button
     Then I should see the Force Change Password screen
+    And I type my password
+    And I confirm my password
+    Given I intercept '{ "headers": { "X-Amz-Target": "AWSCognitoIdentityProviderService.RespondToAuthChallenge" } }' with fixture "force-change-password-phone-success"
+    And I mock 'Amplify.Auth.currentAuthenticatedUser' with fixture "Auth.currentAuthenticatedUser-sms-mfa"
+    Then I click the "Change Password" button
+    Then I see "+1"
```

Comment:
Why is it +1 and not the whole number?

Reply:
The whole number felt like it could be a real number so I had it cut short. 

I'll change it to more unrealistic number.

Reply:
```suggestion
    Then I see "+17755551212"
```

Reply:
👍